src/pages/DataResearch.tsx

```diff
@@ -0,0 +1,227 @@
+import React, { useMemo } from 'react'
+import { useGroupBy } from 'src/api/groupByService'
+import Widget from 'src/shared/Widget'
+import { useDate } from './components/DateTimePicker'
+import moment from 'moment'
+import { Grid } from '@mui/material'
+import { DateSelector } from './components/DateSelector'
+import { useTranslation } from 'react-i18next'
+import { Skeleton } from 'antd'
+import {
+  Area,
+  Tooltip,
+  AreaChart,
+  CartesianGrid,
+  ResponsiveContainer,
+  XAxis,
+  YAxis,
+} from 'recharts'
+import { PageContainer } from './components/PageContainer'
+import { getColorName } from './dashboard/AllLineschart/OperatorHbarChart/OperatorHbarChart'
+
+const now = moment()
+const unique: (value: string, index: number, self: string[]) => boolean = (value, index, self) =>
+  self.indexOf(value) === index
+
+export const DataResearch = () => {
+  return (
+    <PageContainer>
+      <Widget>
+        <h1>מחקרים</h1>
+        <p>אם יש לכם רעיון מעניין למה קורים פה דברים, דברו איתנו בסלאק!</p>
+      </Widget>
+      <StackedResearchSection />
+    </PageContainer>
+  )
+}
+
+function StackedResearchSection() {
```

Comment:
You should move it above the DataResearch component

Reply:
why?

Reply:
As I wrote in different comment you don't have to